recipe/meta.yaml

```diff
@@ -1,27 +1,27 @@
-{% set version = "2.0" %}
+{% set version = "2.1.0" %}
 
 package:
   name: flask-assets
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/F/Flask-Assets/Flask-Assets-{{ version }}.tar.gz
-  sha256: 1dfdea35e40744d46aada72831f7613d67bf38e8b20ccaaa9e91fdc37aa3b8c2
+  sha256: f84d6532ffe59c9ff352885e8740ff4da25c0bcfacd805f0a806815e44354813
 
 build:
   number: 0
   noarch: python
   script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
-  build:
+  host:
     - python
     - pip
   run:
     - python
     - setuptools
     - flask >=0.8
-    - webassets >={{ version}}
+    - webassets >={{ version }}
```

Comment:
webassets 2.1.0 does not exist at this time (see https://pypi.org/project/webassets/#history).

`flask-assets` 2.1.0 depends on `webassets >=2.0`.
```suggestion
    - webassets >=2.0
```
See: https://github.com/miracle2k/flask-assets/blob/2.1.0/setup.py#L53-L56
and: https://github.com/miracle2k/flask-assets/blob/2.1.0/src/flask_assets.py#L23-L25